src/pages/CirclesPage/CirclesPage.tsx

```diff
@@ -54,16 +56,33 @@ export const CirclesPage = () => {
   const isAdmin = (org: QueryResult['organizations'][0]) =>
     org.circles.map(c => c.users[0]).some(u => u && isUserAdmin(u));
 
-  if (query.isLoading || query.isIdle) return <LoadingModal visible />;
+  const updateLogo = async (isUpdated: boolean) => {
+    setUploadingLogo(isUpdated);
```

Comment:
should `isUpdated` be changed to `inProgress` or something? likewise, the name `commit` in OrgLogoUpload is a bit confusing, and I think it should be something like `setInProgress`